content/docs/getting-started/installation/helm/modules/authorizationv2-0.md

```diff
@@ -190,6 +190,103 @@ spec:
   {{</tab >}}
   {{< /tabpane >}}
   {{< /collapse >}}
+{{< collapse id="4" title="SecretProviderClass configuration with array-based, Redis, and config credentials" card="false" >}}
+
+  <br>
+  {{< tabpane Ordinal="4" name="secret-provider-class-with-redis-and-config" lang="bash">}}
```

Comment:
4th step for creating JWT signing K8s secret is optional if user sets up SPC. We have to indicate that.